test/conftest.py

```diff
@@ -12,6 +17,18 @@
 import pytest
 import ssl
 
+
+# By default, tests run against a CQL server (Scylla or Cassandra) listening
+# on localhost:9042. Add the --host and --port options to allow overiding
+# these defaults.
+def pytest_addoption(parser):
+    parser.addoption('--host', action='store', default='localhost',
+                     help='CQL server host to connect to')
+    parser.addoption('--port', action='store', default='9042',
+                     help='CQL server port to connect to')
+    parser.addoption('--ssl', action='store_true',
+                     help='Connect to CQL via an encrypted TLSv1.2 connection')
+
```

Comment:
Now I see you moved this code in this patch, which is confusing to review.
Anyway, my comment in the original place is still relevant - the name "--port" and "--ssl" make sense for CQL tests. Other types of tests will probably need different parameters, and the very general-sounding name "--port" can get confusing.